code/modules/mob/living/emote.dm

```diff
@@ -275,6 +275,16 @@
 	key_third_person = "smugs"
 	message = "grins smugly."
 
+/datum/emote/living/snap

```

Comment:
This emote works for all `mob/living`s. Either move it to only humans or add hilarious variants of `message` for all the types of mobs that can snap. The variables are already there, just redefine them.

Reply:
I detest the emote. If anything keep it to `human` only, not for all mobs.

Reply:
"**Ian** snaps his cute paws. Awwwwww."